app/models/user.rb

```diff
@@ -316,7 +316,13 @@ def content_followed_in_period(start_time, end_time)
     tagnames.each do |tagname|
       node_ids = node_ids + NodeTag.where(tid: tagname.tid).collect(&:nid)
     end
-    Node.where(nid: node_ids).where("created >= #{start_time.to_i}  OR changed >= #{start_time.to_i} AND created <= #{end_time.to_i}  AND changed <= #{end_time.to_i}")
+    
+    Node.where(nid: node_ids)
+        .includes(:revision, :tag)
+        .references(:node_revision)
+        .where("created >= #{start_time.to_i}  OR timestamp >= #{start_time.to_i} AND created <= #{end_time.to_i}  AND timestamp <= #{end_time.to_i}")
```

Comment:
I think the last `AND` should be an `OR` instead.

Reply:
hmm...I found one more bug in query , thanks @seafr I am making it more clearer

Reply:
Thank you for pointing this out. Honestly, I'm not really comfortable with SQL.